docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     build:
       context: ./
       dockerfile: app.dockerfile
+    depends_on:
+      - search
```

Comment:
@TJTorola yea the named instances in the docker compose can become dependents of other instances and it looks like those names are exposed in sort of a local DNS, seems to work with this configuration anyways 😄

Reply:
Nice!